src/sass/components/_header.scss

```
@@ -0,0 +1,205 @@
+.header {
+  .container {
+    @include flex(row, center, space-between);
+  }
+
+  &__top {
+    width: 100%;
+    height: 50px;
+    background-color: $bg-gray;
+
+    &__left ul {
+      display: flex;
+      li {
+        width: 50px;
+        height: 50px;
+        @include flex(center, center);
+        transition: $transition;
+        cursor: pointer;
+        border-left: 1px solid $line-gray;
+        &:last-child {
+          border-right: 1px solid $line-gray;
+        }
+        a {
```

```suggestion
        border-left: 1px solid $line-gray;
        
        &:last-child {
          border-right: 1px solid $line-gray;
        }
        
        a {
```
Check lại file này, mỗi block cách nhau 1 dòng nhé